content/user-management copy.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "User Management"
+metaTitle: "sensenet - User Management"
+metaDescription: "sensenet User Management"
+---
+
+User management is a core function of the business, that includes user creation, rights management and assigning roles to the users. Effective user management contains adjusting the access rights of the users who change role, get promoted or leave the company. It is important to close these changes in short time, this way we can increase employee productivity.
+
+# Users and groups as content
+
+sensenet is built around content, as well as users and groups. Every content is built up by fields, so for example a user as content has numerous special fields as well for storing data like their username and password (among others).
+
+# Built-in groups and users
+
+sensenet offers special and built-in groups and users by default. All users and almost all groups are stored in the /Root/IMS folder. It is possible to create groups outside the IMS folder. These are local groups created for using individual workspaces.
+
+Built-in groups in sensenet are regular groups, meaning their membership can be freely modified. They are used in the default Content Repository structure extensively, but no hard-coded logic is built on them (except for the Administrators group, please do not delete that one). This means you can freely change or replace these groups with your own custom groups in your permission settings. The best practice however is that you build on them, using the suggested purpose in the following sections.
+
+- public domain (/Root/IMS/Public) for the users
+
+- builtin domain (hidden) for technical users and groups that cannot be modified but its capabilities could be used in development
+
+   - Visitor: Everybody is a visitor until logging in. Therefore you cannot log in as a Visitor user because it represents users who are not logged in. It has a couple of permission settings in the default structure for content that should be accessible for everybody. in case you are building a public website with sensenet with a subtree that contains content that should be available to browse and open publicly, you have work with Visitor and allow this technical user to see and open contents where it is necessary.
+   - Everyone: This is the largest group in the system: every single user is considered as a member of the Everyone group, except the Visitor user. This behavior cannot be changed. Please set permissions for this group carefully, as every logged in user will have access to those content. Note that you cannot set deny permissions for the Everyone group, because it would make that content inaccessible for - well, for everyone, including you. Everyone group may come in handy when you want to allow something to be used and see by every logged-in user the same way.
```

Comment:
```suggestion
   - Everyone: This is the largest group in the system: every single user is considered as a member of the Everyone group, except the Visitor user. This behavior cannot be changed. Please set permissions for this group carefully, as every logged in user will have access to those content. Note that you cannot set deny permissions for the Everyone group, because it would make that content inaccessible for - well, for everyone, including you. Everyone group may come in handy when you want to allow something to be used and seen by every logged-in user the same way.
```